Modules/Devices/src/Devices.Domain/Aggregates/Tier/TierName.cs

```diff
@@ -7,6 +7,7 @@ public record TierName
 {
     public string Value { get; }
     public const int MAX_LENGTH = 30;
+    public const int MINIMUM_LENGTH = 3;
```

Comment:
Two minor things:

1. Can you move MINIMUM_LENGTH above MAX_LENGTH please? 
2. And please use either MIN/MAX or MINIMUM/MAXIMUM, but not a combination of both. :)